include/deal.II/matrix_free/dof_info.h

```diff
@@ -241,6 +241,14 @@ namespace internal
         const std::vector<FaceToCellTopology<1>> &ghosted_faces,
         const bool                                fill_cell_centric);
 
+      /**
+       * TODO
+       */
+      void
+      compress_for_shared_memory_usage(
```

Comment:
What would be a better name?

Reply:
How about `identify_shared_memory_ranges` or `_regions`?